commitlint.config.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
```

Comment:
Adding a weird comment here:

![image](https://user-images.githubusercontent.com/171934/53518999-3af7af80-3aa0-11e9-9006-023192f768ba.png)

GitHub won't let me comment on that file because there's no lines in it ... and it doesn't really indicate what it is. Is it a symlink to allow for referencing `@silvermine/eslint-config/commitlint` without needing to reference the `*.config.js` file specifically?

Reply:
That shouldn't bee there. I must have created it by accident somehow. I removed it.